src/webhook/validation/edgeconnect/validation.go

```diff
@@ -0,0 +1,81 @@
+package edgeconnect
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/Dynatrace/dynatrace-operator/src/api/v1alpha1/edgeconnect"
+	"github.com/Dynatrace/dynatrace-operator/src/scheme"
+	"github.com/Dynatrace/dynatrace-operator/src/webhook/validation"
+	"github.com/pkg/errors"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+type edgeconnectValidator struct {
+	clt       client.Client
+	apiReader client.Reader
+	cfg       *rest.Config
+}
+
+func newEdgeConnectValidator(apiReader client.Reader, cfg *rest.Config) admission.Handler {
+	return &edgeconnectValidator{
+		apiReader: apiReader,
+		cfg:       cfg,
+	}
+}
+
+func AddEdgeConnectValidationWebhookToManager(manager ctrl.Manager) error {
+	manager.GetWebhookServer().Register("/validate-edgeconnect", &webhook.Admission{
```

Comment:
Do we want to adapt the path of the dynakube validator to `"/validate-dynakube"`? 

FYI: Apart from this question I would approve this PR as I tested it and it works.

Reply:
fixed